addon/components/bread-crumb.js

```diff
@@ -14,6 +14,7 @@ export default Component.extend({
   layout,
   tagName: 'li',
   classNameBindings: ['crumbClass'],
+  i18n: false,
```

Comment:
Based on the current implementation, I don't see why any change is needed to this lib.  `title` can be the translation key and you can implement the block form of the component.

```
{{#bread-crumbs outputStyle="bootstrap" linkable=true as |component crumb|}}
  {{#bread-crumb route=crumb breadCrumbs=component}}
      {{t crumb.title}}
  {{/bread-crumb}}
{{/bread-crumbs}}
```

I haven't used this library, it's just my initial reaction looking at the public API.